### ISSUE TITLE
Error on cold start firestore-send-email

### ISSUE THREAD

**Issue by rphlmr** [firebase/extensions#48]
### [READ] Step 1: Are you in the right place?

Issues filed here should be about bugs for a **specific extension in this repository**.
If you have a general question, need help debugging, or fall into some
other category use one of these other channels:

- For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/)
  with the firebase tag.
- For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk)
  google group.
- To file a bug against the Firebase Extensions platform, or for an issue affecting multiple extensions, please reach out to
  [Firebase support](https://firebase.google.com/support/troubleshooter/contact/) directly.

### [REQUIRED] Step 2: Describe your configuration

- Extension name: `firestore-send-email`
- Configuration values (redact info where appropriate):
  - all config require (extension is working, except after a cold start)

#### Steps to reproduce:

firestore-send-email throw an unhandled error at cold start : 

```
Unhandled error occurred during processing: { Error
    at Http2CallStream.call.on (/srv/node_modules/@grpc/grpc-js/build/src/client.js:96:45)
    at emitOne (events.js:121:20)
    at Http2CallStream.emit (events.js:211:7)
    at process.nextTick (/srv/node_modules/@grpc/grpc-js/build/src/call-stream.js:71:22)
    at _combinedTickCallback (internal/process/next_tick.js:132:7)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
  code: 13,
  details: '',
  metadata: Metadata { options: undefined, internalRepr: Map {} },
  note: 'Exception occurred in retry method that was not classified as transient' }
```

##### Expected result

Mail send / or send retry

##### Actual result

Mail not sent and not retried

**Comment by Ehesp** [comment]
Thanks for the report! @laurenzlong would you be able to help take a look at this?

**Comment by rphlmr** [comment]
I think it'll be linked to https://github.com/firebase/firebase-functions/issues/536

A problem in the grpc implementation.

In my own functions I have the same problem and I rollback dependencies to  :  

```
"firebase-admin": "7.4.0",
"firebase-functions": "2.3.1"
```

**Comment by laurenzlong** [comment]
Thanks for the helpful issue link @rphlmr! The best course of action is to implement the suggestion in https://github.com/firebase/firebase-functions/issues/536#issuecomment-531923813. I confirmed with Sebastian, author of that comment, who works on Firestore, and he confirmed that was the most reliable thing to do at the moment. This bug also affects not just firestore-send-email, but other extensions that write to Firestore also. I'll make a PR for this.

**Comment by rphlmr** [comment]
Hi :) 
Thanks for the PR.
I have a stupid question, excuse me, but do we'll receive an email when updates are released ?

**Comment by laurenzlong** [comment]
Hi @rphlmr if you would like to be informed of new releases, you can click the "watch" button at the top right corner of this page, and then select "releases only".

**Comment by rphlmr** [comment]
Thanks everyone for the 0.1.1 ;)

**Comment by laurenzlong** [comment]
You're welcome! Thanks for reporting!

**Comment by licalac** [comment]
Hi everyone,

I'm using Trigger mail Version 0.1.2 and just have same issue today ...

mail-processQueue - Unhandled error occurred during processing: { Error at Http2CallStream.call.on (/srv/node_modules/@grpc/grpc-js/build/src/client.js:96:45) at emitOne (events.js:121:20) at Http2CallStream.emit (events.js:211:7) at process.nextTick (/srv/node_modules/@grpc/grpc-js/build/src/call-stream.js:71:22) at _combinedTickCallback (internal/process/next_tick.js:132:7) at process._tickDomainCallback (internal/process/next_tick.js:219:9) code: 13, details: '', metadata: Metadata { options: undefined, internalRepr: Map {} }, note: 'Exception occurred in retry method that was not classified as transient' }
-- | --

that was not supposed to be fixed since November on version 0.1.1 ?

**Comment by fgatti675** [comment]
Got the same with 0.1.2:
`Unhandled error occurred during processing: { Error
    at Http2CallStream.call.on (/srv/node_modules/@grpc/grpc-js/build/src/client.js:96:45)
    at emitOne (events.js:121:20)
    at Http2CallStream.emit (events.js:211:7)
    at process.nextTick (/srv/node_modules/@grpc/grpc-js/build/src/call-stream.js:71:22)
    at _combinedTickCallback (internal/process/next_tick.js:132:7)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
  code: 13,
  details: '',
  metadata: Metadata { options: undefined, internalRepr: Map {} },
  note: 'Exception occurred in retry method that was not classified as transient' }`
